website/content/en/highlights/2021-08-24-helm-vector-dev.md

```diff
@@ -0,0 +1,30 @@
+---
+date: "2021-08-24"
+title: "Introducing https://helm.vector.dev"
+description: "A new home for Vector's Helm charts"
+authors: ["spencergilbert"]
+pr_numbers: []
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "deprecation"
+  platforms: ["helm"]
+---
+
+Vector’s 0.16.0 release will be the last version that publishes charts to both
```

Comment:
Maybe we should say `0.16.X` in case we have patch releases that we do publish the charts for.

Reply:
🤷 sure - patch releases seem reasonable. In that case it's more like 0.17 will not release charts, rather than 0.16 is the _last_ release